src/modules/store.js

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable quotes */
+const InnerTodo = document.querySelector(".todo-inner");
+let isEdit = false;
+let editId = null;
+
+export default class display {
+  static getTodo = () => {
+    let Todo;
+    if (localStorage.getItem("todo") == null) {
+      Todo = [];
+    } else {
+      Todo = JSON.parse(localStorage.getItem("todo"));
+    }
+    return Todo;
+  };
+
+  static addRemoveEvent = () => {
+    const trash = document.querySelectorAll(".trash");
+    trash.forEach((task, i) => {
+      task.addEventListener("click", (ev) => {
+        ev.preventDefault();
+        this.removeTodo(i);
+      });
+    });
+  };
+
+  static removeTodo = (id) => {
+    const toDos = display.getTodo();
+    toDos.splice(id, 1);
+    toDos.forEach((item) => {
+      if (item.index > id) {
+        item.index -= 1;
```

Comment:
- [ ] `The property index set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5).` This requirement is not fulfilled at all, the indexes in the array are zero based which should be based on the 1, that means the first item should have index = 1, isntead of being zero